frontend/lib/src/components/elements/CodeBlock/CopyButton.tsx

```diff
@@ -14,9 +14,10 @@
  * limitations under the License.
  */
 
-import Clipboard from "clipboard"
 import React, { PureComponent, ReactNode, createRef } from "react"
+import Clipboard from "clipboard"
```

Comment:
Seems like a weird change...shouldn't Clipboard be above for alphabet?

Reply:
I thought `React` imports should always be on top. At least I have the feeling that its the case in most of our files. But I think we should anyways introduce something that auto orders our frontend imports after some rules.